schema/src/main/java/org/hl7/v3/ObjectFactory.java

```diff
@@ -73,6 +73,9 @@
     private final static QName _StrucDocItemTable_QNAME = new QName("urn:hl7-org:v3", "table");
 
     private final static QName _CD_QNAME = new QName("urn:hl7-org:v3", "code");
+    private final static QName _VALUE_QNAME = new QName("urn:hl7-org:v3", "value");
+    private final static QName _ObservationStatement_QNAME = new QName("urn:hl7-org:v3", "ObservationStatement");
```

Comment:
Is this ObservationStatement still being used for testing? If not remember to remove the JAXB Element below aswell